Calendars/Calendars.Plugin.UWP/CalendarsImplementation.cs

```diff
@@ -411,8 +410,12 @@ public async Task<bool> DeleteEventAsync(Calendar calendar, CalendarEvent calend
 
 #region Private Methods
 
+        [MemberNotNull(nameof(_apptStore), nameof(_localApptStore))]
         private async Task EnsureInitializedAsync()
         {
+            // If EnsureInitializedAsync is not awaited, they won't be initialized. It's fine.
+#pragma warning disable CS8774 // Member must have a non-null value when exiting.
```

Comment:
`async` initialization works, `sync` - isn't. As long as `EnsureInitializedAsync` is always awaited, everything should work fine.

Reply:
Yeah I meant that I was surprised the compiler didn't detect that and enforce `MemberNotNull` asynchronously... though I guess that could be a hard problem...

Reply:
That's a good point. ~~I guess that's because this attribute is designed to be used by constructors: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis#constructor-helper-methods-membernotnull-and-membernotnullwhen~~
Nope this attribute is intended for exactly this purpose: https://github.com/dotnet/runtime/issues/31877

Having `MemberNotNull` work with async methods would be a great improvement, I agree.

Reply:
Created an issue: https://github.com/dotnet/runtime/issues/56356